.github/workflows/_ci.yaml

```diff
@@ -251,6 +280,19 @@ jobs:
     with:
       IMAGE: ${{ needs.build-levanter.outputs.DOCKER_TAG_FINAL }}
       TEST_NAME: levanter
+      EXECUTE: |
+        docker run --gpus all --shm-size=1g ${{ needs.build-levanter.outputs.DOCKER_TAG_FINAL }} bash -ec "pip install flake8 pytest test-util && pytest /opt/levanter/tests" | tee test-levanter.log
+      STATISTICS_SCRIPT: |
+        errors=$(cat test-*.log | grep -c 'ERROR:' || true)
+        failed_tests=$(cat test-*.log | grep -c 'FAILED in' || true)
```

Comment:
This likely still suffers from the failure in:
https://github.com/NVIDIA/JAX-Toolbox/actions/runs/7981652065/job/21794674559
https://github.com/NVIDIA/JAX-Toolbox/actions/runs/7981652065/job/21794674761

Reply:
Fixed